ui/dashboard/views/index.ejs

```diff
@@ -0,0 +1,162 @@
+					everyone you know!
+				</div>
+				<div class="social-icons">
+					<a
+						target="_blank"
+						href="https://www.facebook.com/sharer.php?s=100&p[url]=https%3A%2F%2Ffoldforcovid.io"
+						><img src="views/facebook.svg"
+					/></a>
+					<a
+						target="_blank"
+						href="https://twitter.com/share?url=https%3A%2F%2Ffoldforcovid.io&text=I%E2%80%99m%20fighting%20%23coronavirus%20using%20my%20Raspberry%20Pi%20or%20old%20laptop%2C%20visit%20foldforcovid.io%20to%20set%20up%20your%20own%20and%20join%20the%20fight!%20%23foldforcovid%20%40RosettaAtHome%20%40balena_io"
+						><img src="views/twitter.svg"
+					/></a>
+					<a
+						target="_blank"
+						href="https://www.linkedin.com/shareArticle/?url=https%3A%2F%2Ffoldforcovid.io&mini=true"
+						><img src="views/linkedin.svg"
+					/></a>
+				</div>
+			</div>
+		</nav>
+		<div id="account_key_modal">
+			<div class="overlay"></div>
+			<div class="modal">
+				<form id="account_key_form">
+					<div class="form-control">
+						<label for="accountKey">
+							Account Key
+						</label>
+						<input type="text" name="accountKey" palceholder="Account key" />
+					</div>
+					<div class="form-control">
+						<button type="submit" id="account_key_submit">
+							Apply
+						</button>
+					</div>
+				</form>
+			</div>
+		</div>
+		<div id="shutdown_loader">
+			<div class="overlay">
+				<div class="spinner">
+					<div class="dot1"></div>
+					<div class="dot2"></div>
+				</div>
+				<span class="loader-title">Device is powering off</span>
+			</div>
+		</div>
+		<div id="reboot_loader">
+			<div class="overlay">
+				<div class="spinner">
+					<div class="dot1"></div>
+					<div class="dot2"></div>
+				</div>
+				<span class="loader-title">Device is rebooting</span>
+			</div>
+		</div>
+		<iframe
+			id="tty"
+			width="100%"
+			height="90%"
+			frameborder="0"
+			onload="if(document.getElementById('tty').src == '') { document.getElementById('tty').src = window.location.protocol + '//' + window.location.hostname + ':8080/'; }"
+		></iframe>
+
+		<script>
+			(function () {
+				window.onload = function () {
+					var accountKeyModal = document.getElementById("account_key_modal");
+					var rebootLoader = document.getElementById("reboot_loader");
+					var shutdownLoader = document.getElementById("shutdown_loader");
+					var shutdownButton = document.getElementById("shutdown");
+					var rebootButton = document.getElementById("reboot");
+					var accountKeyButton = document.getElementById("account_key");
+					var accountKeyForm = document.getElementById("account_key_form");
+
+					shutdownButton.onclick = function (event) {
+						fetch("/manage/shutdown", { method: "post" });
+						shutdownLoader.style.display = "block";
+					};
+
+					rebootButton.onclick = function (event) {
+						fetch("/manage/reboot", { method: "post" });
+						rebootLoader.style.display = "block";
+
+						window.setTimeout(function () {
```

Comment:
Why do we have this for reboot, but not shutdown?

Reply:
Not Sure, @chrisys wanted the loader only for 60 sec for reboot and not for shutdown

Reply:
My thinking was if the device has shutdown it's not coming back so we didn't need to time it out.

Reply:
We still show a spinner in the dashboard, and I think it still responds before it shuts down (once all processes are killed, etc.)? I am fine either way, just curious.